DockerizeJenkinsfile

```diff
@@ -7,7 +7,7 @@ def use_release_artifacts() {
 pipeline {
     agent {
         docker {
-            image 'registry.gear.ge.com/dig-predix-security-services/uaa-ci-testing:latest'
+            image 'dig-grid-artifactory.apps.ge.com/pgog-fss-iam-uaa-docker-stage/uaa-ci-testing:latest'
```

Comment:
This file and corresponding pipeline are outdated and not used any more. Let's leave this file as is.

Reply:
Done, reverted changes.